docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>we get 5 objects from s3 but are only using 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>yes, maybe we could show how to merge into the same dataframe? hopefully, they have a unified schema.</p>